src/classes/AFFL_AccRecordType_TDTM.cls

```diff
@@ -48,6 +51,12 @@ public class AFFL_AccRecordType_TDTM extends TDTM_Runnable {
         TDTM_Runnable.Action triggerAction, Schema.DescribeSObjectResult objResult) {
 
     	 DmlWrapper dmlWrapper = new DmlWrapper();
+
+         if (TDTM_ProcessControl.getRecursionFlag(TDTM_ProcessControl.registeredTrigger.AFFL_AccRecordType_TDTM)) {
+            reentrancyPrevented = true;
```

Comment:
I don't necessarily like doing this, but it's the only way I know of to easily test that the reentrancy check works as expected in a unit test.